third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -513,12 +535,22 @@ function getStackFrames() {
 function buildRrpObjectResult(cdpReturnValue) {
   const rrpResult = { data: {} };
   if (cdpReturnValue) {
-    const { result: cdpResult, exceptionDetails } = cdpReturnValue;
-    const rrpId = buildRrpObjectFromCdpObject(cdpResult);
-
+    const { result: cdpResult, exceptionDetails, commandFailedReason } = cdpReturnValue;
     if (exceptionDetails) {
+      // exceptionDetails encapsulate a JS exception thrown while handling the command.
+      const rrpId = buildRrpObjectFromCdpObject(exceptionDetails);
       rrpResult.exception = rrpId;
-    } else {
+    }
+    else if (commandFailedReason) {
+      // commandFailedReason is a plain object, containing `code` and `message`, indicating 
+      // why the V8 debugger failed to handle the command.
+      const rrpId = registerPlainObject(commandFailedReason);
+      rrpResult.exception = rrpId;
```

Comment:
I think reusing `exception` for this is mixing concerns in a way that is a bad idea, because it mixes user-level code exceptions with runtime-level implementation exceptions.  With this a user will see a random exception that has nothing to do with their code, I'm guessing. It would actually depend on lots of details that I don't know off the top of my head.

I don't know how best to foster intra-team communication around this stuff, but it feels important that we do so, since the backend and runtime both rely heavily on the behavior of this protocol interface.